lib/auth/auth.go

```diff
@@ -910,6 +921,29 @@ func (a *Server) updateVersionMetrics() {
 	}
 }
 
+// refreshRemoteClusters updates connection status of all remote clusters.
+func (a *Server) refreshRemoteClusters(ctx context.Context) {
+	log.Debug("Refreshing remote cluster statuses.")
+
+	remoteClusters, err := a.Services.GetRemoteClusters()
+	if err != nil {
+		log.Errorf("Failed to load remote clusters for status refresh: %v", err)
```

Comment:
```suggestion
		log.WithError(err).Error("Failed to load remote clusters for status refresh")
```

Same for others.

Reply:
Right. I saw both styles in this file. Will make a note that structured approach is preferred.

Fixed in https://github.com/gravitational/teleport/pull/22455/commits/1434fb47c41ea7a5d9dc962f5f3db1a5f7e7a848

Reply:
Style is a bit all-over (I'm guilty of both too), but long-term I think structured is what we want - it's bound to be better for Cloud (and various self-hosted folks too).

Reply:
Looks like using `%v` causes the stacktrace to appear in the logs, so my update to `WithError(err)` should still make sense.